### ISSUE TITLE
[Issue]: Flac Transcoding (for Chromecast) is broken.

### ISSUE THREAD

**Issue by kevincox** [jellyfin/jellyfin#8722]
### Please describe your bug

Trying to play a FLAC file on chromecast fails because it tries to transcode the file and does something *very strange*.

Steps to reproduce:
1. Connect to a chromecast.
2. Play a FLAC file with a high enough bitrate to (incorrectly, but that is a different bug) trigger transcoding.

Notably is it trying to load: https://jellyfin.example/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/master.m3u8?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit. That URL gives:

```
HTTP/1.1 200 
server: nginx
date: Wed, 09 Nov 2022 21:39:31 GMT
content-type: application/vnd.apple.mpegurl
content-length: 470
expires: 0
x-response-time-ms: 3

#EXTM3U
#EXT-X-STREAM-INF:BANDWIDTH=128000,AVERAGE-BANDWIDTH=128000,CODECS="fLaC"
main.m3u8?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit
```

That then gives:

```
HTTP/2 200
server: nginx
date: Wed, 09 Nov 2022 21:42:23 GMT
content-type: application/vnd.apple.mpegurl
content-length: 87432
x-response-time-ms: 4

#EXTM3U
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:3
#EXT-X-MEDIA-SEQUENCE:0
#EXTINF:3.000000, nodesc
hls1/main/0.ts?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit&runtimeTicks=0&actualSegmentLengthTicks=30000000
...
hls1/main/182.ts?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit&runtimeTicks=5460000000&actualSegmentLengthTicks=19831552
#EXT-X-ENDLIST
```

Up to this point it seems somewhat reasonable. It is creating a playlist to stream the file in chunks. However the file referenced here is completely broken. Here is what attempting to play it results in:

```
% ffprobe 'https://jellyfin.example/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/hls1/main/0.ts?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit&runtimeTicks=0&actualSegmentLengthTicks=30000000'
ffprobe version 5.1.2 Copyright (c) 2007-2022 the FFmpeg developers
  built with gcc 11.3.0 (GCC)
  configuration: --disable-static --prefix=/nix/store/zv91f2858fcyv0vmip2f665m92hs73cw-ffmpeg-full-5.1.2 --target_os=linux --arch=x86_64 --enable-gpl --enable-version3 --disable-nonfree --enable-shared --enable-pic --disable-small --enable-runtime-cpudetect --disable-lto --enable-gray --enable-swscale-alpha --enable-hardcoded-tables --enable-safe-bitstream-reader --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avutil --enable-postproc --enable-swresample --enable-swscale --enable-doc --disable-htmlpages --enable-manpages --disable-podpages --disable-txtpages --enable-bzlib --enable-libcelt --enable-cuda --enable-cuda-llvm --enable-libdav1d --disable-libfdk-aac --disable-libflite --enable-fontconfig --enable-libfreetype --enable-frei0r --enable-libfribidi --enable-libgme --enable-gnutls --enable-libgsm --enable-ladspa --enable-libmp3lame --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdc1394 --enable-libdrm --enable-iconv --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmysofa --enable-libopus --enable-librsvg --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-vaapi --enable-vdpau --enable-libvorbis --enable-libvmaf --enable-libvpx --enable-libwebp --enable-xlib --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libxml2 --enable-lzma --enable-cuvid --enable-nvdec --enable-nvenc --enable-openal --enable-opencl --enable-libopencore-amrnb --disable-opengl --enable-libopenh264 --enable-libopenjpeg --disable-openssl --enable-libpulse --enable-librav1e --enable-libsvtav1 --disable-librtmp --enable-sdl2 --enable-libsoxr --enable-libspeex --enable-libvidstab --enable-libvo-amrwbenc --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzimg --enable-zlib --enable-vulkan --enable-libglslang --enable-libsmbclient --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
  libavdevice    59.  7.100 / 59.  7.100
  libavfilter     8. 44.100 /  8. 44.100
  libswscale      6.  7.100 /  6.  7.100
  libswresample   4.  7.100 /  4.  7.100
  libpostproc    56.  6.100 / 56.  6.100
[mpegts @ 0x1cb4540] start time for stream 0 is not set in estimate_timings_from_pts
[mpegts @ 0x1cb4540] stream 0 : no TS found at start of file, duration not set
[mpegts @ 0x1cb4540] Could not find codec parameters for stream 0 (Video: h264 ([27][0][0][0] / 0x001B), none): unspecified size
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
Input #0, mpegts, from 'https://jellyfin.example/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/hls1/main/0.ts?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit&runtimeTicks=0&actualSegmentLengthTicks=30000000':
  Duration: 00:09:07.97, start: 10.000000, bitrate: 4754 kb/s
  Program 1 
    Metadata:
      service_name    : Service01
      service_provider: FFmpeg
  Stream #0:0[0x100]: Video: h264 ([27][0][0][0] / 0x001B), none, 90k tbr, 90k tbn
  Stream #0:1[0x101]: Data: bin_data ([6][0][0][0] / 0x0006)
Unsupported codec with id 98314 for input stream 1
```

It has a video stream (for a FLAC file!) and a broken stream.

Not only that but the file is 311MiB. This is larger than the entire file (which is 302MiB)

Further stranger, if we download a few of these files the are all identical.

```
% sha1sum /tmp/*.ts
4db8b600a517b6c0bd4339405933e221db365239  /tmp/0.ts
4db8b600a517b6c0bd4339405933e221db365239  /tmp/10.ts
4db8b600a517b6c0bd4339405933e221db365239  /tmp/1.ts
```

So something is very broken here with flac transcoding.

### Jellyfin Version

10.8.0

### if other:

10.8.7

### Environment

```markdown
- OS: Linux
- Virtualization: None
- Clients: Chromecast (multiple devices, ultra, home mini, third party)
- Browser: N/A
- FFmpeg Version: 5.1.2
- Playback Method: App to chromecast
- Hardware Acceleration: None
- Plugins: None
- Reverse Proxy: nginx
- Base URL: private
- Networking: Bridge
- Storage: local
```

### Jellyfin logs

```shell
Here are jellyfin + nginx logs:


{"cache_status":"","connection_id":"17851","connection_request_count":"7","duration_s":"0.013","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"140","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"GET","req_pipelined":".","req_referrer":"","req_uri":"/Playlists/17570cd1d54aa6f9c6ce3f85e50e745c/Items?Fields=PrimaryImageAspectRatio&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&UserId=8acde645a7ed4ab6936044a27cf5eb7c","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (Linux; Android 12; Nexus 5 Build/SQ3A.220705.004; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/107.0.5304.91 Mobile Safari/537.36","res_bytes":"3304","res_bytes_body":"3183","res_content_length":"","res_content_type":"application/json; charset=utf-8","res_status":"200","res_gzip_ratio":"7.28","ssl_session_id":"8366b3732eae21d103185e0eade8f86c07953d1dc372b50897c6d254d7a6faeb","ssl_session_reused":"r","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Playlists/17570cd1d54aa6f9c6ce3f85e50e745c/Items?Fields=PrimaryImageAspectRatio&EnableImageTypes=Primary%2CBackdrop%2CBanner%2CThumb&UserId=8acde645a7ed4ab6936044a27cf5eb7c"}
{"cache_status":"","connection_id":"17334","connection_request_count":"325","duration_s":"0.002","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"79","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"OPTIONS","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"144","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3"}
{"cache_status":"","connection_id":"17334","connection_request_count":"326","duration_s":"0.003","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"72","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"GET","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"1659","res_bytes_body":"1516","res_content_length":"","res_content_type":"application/json; charset=utf-8","res_status":"200","res_gzip_ratio":"2.58","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3"}
{"cache_status":"","connection_id":"17334","connection_request_count":"327","duration_s":"0.001","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"123","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"OPTIONS","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Items/cd3e80ef39a9a5b450f45ea673e90eb3/PlaybackInfo?UserId=8acde645a7ed4ab6936044a27cf5eb7c&StartTimeTicks=0&MaxStreamingBitrate=3000000","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"154","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Items/cd3e80ef39a9a5b450f45ea673e90eb3/PlaybackInfo?UserId=8acde645a7ed4ab6936044a27cf5eb7c&StartTimeTicks=0&MaxStreamingBitrate=3000000"}
Nov 09 21:33:23 t6 jellyfin[1386]: [21:33:23] [INF] [53] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for Kevin. EnableAudioPlaybackTranscoding: True
{"cache_status":"","connection_id":"17334","connection_request_count":"328","duration_s":"0.008","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"4118","req_content_length":"4000","req_content_type":"application/json","req_host":"jellyfin.example","req_method":"POST","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Items/cd3e80ef39a9a5b450f45ea673e90eb3/PlaybackInfo?UserId=8acde645a7ed4ab6936044a27cf5eb7c&StartTimeTicks=0&MaxStreamingBitrate=3000000","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"1120","res_bytes_body":"977","res_content_length":"","res_content_type":"application/json; charset=utf-8","res_status":"200","res_gzip_ratio":"2.10","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Items/cd3e80ef39a9a5b450f45ea673e90eb3/PlaybackInfo?UserId=8acde645a7ed4ab6936044a27cf5eb7c&StartTimeTicks=0&MaxStreamingBitrate=3000000"}
{"cache_status":"","connection_id":"17334","connection_request_count":"329","duration_s":"0.002","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"43","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"OPTIONS","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing/Progress","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"154","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:23+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing/Progress"}
{"cache_status":"","connection_id":"17334","connection_request_count":"332","duration_s":"0.000","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"37","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"OPTIONS","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"154","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:24+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing"}
Nov 09 21:33:25 t6 jellyfin[1386]: [21:33:25] [WRN] [26] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/Sessions/Playing/Progress to 76.64.94.41 in 0:00:02.0795682 with Status Code 204
{"cache_status":"","connection_id":"17334","connection_request_count":"330","duration_s":"2.081","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"348","req_content_length":"306","req_content_type":"application/json","req_host":"jellyfin.example","req_method":"POST","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing/Progress","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"82","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:25+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing/Progress"}
Nov 09 21:33:25 t6 jellyfin[1386]: [21:33:25] [WRN] [98] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/master.m3u8?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit to 76.64.94.41 in 0:00:02.342904 with Status Code 200
{"cache_status":"","connection_id":"17334","connection_request_count":"331","duration_s":"2.344","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"329","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"GET","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/master.m3u8?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"598","res_bytes_body":"470","res_content_length":"470","res_content_type":"application/vnd.apple.mpegurl","res_status":"200","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:25+00:00","http2":"h2","uri":"https://jellyfin.example/audio/cd3e80ef-39a9-a5b4-50f4-5ea673e90eb3/master.m3u8?DeviceId=chromecast_1668028338040&MediaSourceId=cd3e80ef39a9a5b450f45ea673e90eb3&AudioBitrate=192000&PlaySessionId=cd7fc1fe1adb4ab8abae487587ea2067&api_key=REDACTED&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=362f57a4c7bbb2e872c5ad8ef1c859d4&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&TranscodeReasons=ContainerBitrateExceedsLimit"}
Nov 09 21:33:25 t6 jellyfin[1386]: [21:33:25] [WRN] [99] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/Sessions/Playing to 76.64.94.41 in 0:00:01.6912982 with Status Code 204
{"cache_status":"","connection_id":"17334","connection_request_count":"333","duration_s":"1.692","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"338","req_content_length":"306","req_content_type":"application/json","req_host":"jellyfin.example","req_method":"POST","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"82","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:25+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing"}
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [WRN] [40] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/Sessions/Playing/Progress to 76.64.94.41 in 0:00:01.956929 with Status Code 204
{"cache_status":"","connection_id":"17334","connection_request_count":"334","duration_s":"1.958","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"344","req_content_length":"306","req_content_type":"application/json","req_host":"jellyfin.example","req_method":"POST","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing/Progress","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"82","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:26+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing/Progress"}
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [INF] [99] Jellyfin.Plugin.KodiSyncQueue.EntryPoints.UserSyncNotification: Started user data sync
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [WRN] [26] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/Sessions/Playing/Progress to 76.64.94.41 in 0:00:00.5827976 with Status Code 204
{"cache_status":"","connection_id":"17334","connection_request_count":"335","duration_s":"0.584","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"344","req_content_length":"306","req_content_type":"application/json","req_host":"jellyfin.example","req_method":"POST","req_pipelined":".","req_referrer":"https://apps.jellyfin.org/chromecast/stable/","req_uri":"/Sessions/Playing/Progress","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.121 Safari/537.36 CrKey/1.52.272222","res_bytes":"82","res_bytes_body":"0","res_content_length":"","res_content_type":"","res_status":"204","res_gzip_ratio":"","ssl_session_id":"761b07b6eae618ca7d6effed53d42da076fa146dd6115f1dc4e1b04a4e75777d","ssl_session_reused":".","time":"2022-11-09T21:33:26+00:00","http2":"h2","uri":"https://jellyfin.example/Sessions/Playing/Progress"}
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [WRN] [102] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from https://jellyfin.example/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3 to 76.64.94.41 in 0:00:02.3777384 with Status Code 200
{"cache_status":"","connection_id":"17851","connection_request_count":"8","duration_s":"2.378","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"69","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"GET","req_pipelined":".","req_referrer":"","req_uri":"/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (Linux; Android 12; Nexus 5 Build/SQ3A.220705.004; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/107.0.5304.91 Mobile Safari/537.36","res_bytes":"1638","res_bytes_body":"1516","res_content_length":"","res_content_type":"application/json; charset=utf-8","res_status":"200","res_gzip_ratio":"2.58","ssl_session_id":"8366b3732eae21d103185e0eade8f86c07953d1dc372b50897c6d254d7a6faeb","ssl_session_reused":"r","time":"2022-11-09T21:33:26+00:00","http2":"h2","uri":"https://jellyfin.example/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3"}
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [INF] [99] Jellyfin.Plugin.KodiSyncQueue.EntryPoints.UserSyncNotification: User Data Sync: User Kevin (8acde645a7ed4ab6936044a27cf5eb7c) posted 2 updates
Nov 09 21:33:26 t6 jellyfin[1386]: [21:33:26] [INF] [99] Jellyfin.Plugin.KodiSyncQueue.EntryPoints.UserSyncNotification: Finished user data sync, taking 00:00:00.2677724
{"cache_status":"","connection_id":"17851","connection_request_count":"9","duration_s":"0.004","protocol":"HTTP/2.0","rate_limit":"","remote_addr":"::ffff:76.64.94.41","req_bytes":"69","req_content_length":"","req_content_type":"","req_host":"jellyfin.example","req_method":"GET","req_pipelined":".","req_referrer":"","req_uri":"/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3","req_via":"","req_user":"","req_user_agent":"Mozilla/5.0 (Linux; Android 12; Nexus 5 Build/SQ3A.220705.004; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/107.0.5304.91 Mobile Safari/537.36","res_bytes":"1636","res_bytes_body":"1516","res_content_length":"","res_content_type":"application/json; charset=utf-8","res_status":"200","res_gzip_ratio":"2.58","ssl_session_id":"8366b3732eae21d103185e0eade8f86c07953d1dc372b50897c6d254d7a6faeb","ssl_session_reused":"r","time":"2022-11-09T21:33:26+00:00","http2":"h2","uri":"https://jellyfin.example/Users/8acde645a7ed4ab6936044a27cf5eb7c/Items/cd3e80ef39a9a5b450f45ea673e90eb3"}
```
```


### FFmpeg logs

_No response_

### Please attach any browser or client logs here

_No response_

### Please attach any screenshots here

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kevincox** [comment]
This is likely the same as https://github.com/jellyfin/jellyfin/issues/2279. Maybe it is better to reopen that issue.

**Comment by kevincox** [comment]
Fuck off bot

**Comment by kevincox** [comment]
Go away

**Comment by kevincox** [comment]
Go away

**Comment by kevincox** [comment]
This appears to have been fixed in 10.9.1!
